code/src/index.css

```diff
@@ -1,13 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Roboto', 'Segoe UI', 'Oxygen', 'Ubuntu', 'Cantarell',
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', -apple-system,
+    BlinkMacSystemFont, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #a3d2ca;
+  color: #056676;
+  /* A way for the body to take up the whole screen size (100%) when the content isn't enough to do that */
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

Comment:
First I had it like this but when I look at my page on a mobile device I didn't see all my content since the page took more height than 100vh.

With some support I came up with this solution.

Now when looking at it again, I'm thinking of if I should take some time and try again if it could be solved with your suggestion. Perhaps I will take some time during next week and ask you for some support.